Gemfile.lock

```diff
@@ -333,10 +333,10 @@ DEPENDENCIES
   tzinfo-data
   uglifier
   web-console (<= 4.0.1)
-  webpacker (< 4.0)
+  webpacker
 
 RUBY VERSION
-   ruby 2.6.3p62
+   ruby 2.6.5p114
```

Comment:
Er det påkrævet at køre en nyere Ruby for at køre den nye version af Rails webpacker?

Reply:
Hm, næh. Gemfile.lock må have manglet en opdatering af det versions-nummer.